website/src/categories/components/tag/accessibility.md

```diff
@@ -9,15 +9,15 @@ eleventyNavigation:
 
 ## Keyboard interactions
 
-Users can move through the tags within a tag-list using the `arrow keys`.
-You can navigate back to the previous tag with `left` or `up`. The focus indicator loops, so when you are at the last option and press `down` it will focus on the first tag.
-In the stacked version of tag-list, when there are hidden tags, you can navigate only through visible tags with arrow keys, and you can focus tag with amount of hidden tags by `Tab` key.
+Users can move through the tags within a tag-list using the left and right arrow keys.
+You can navigate back to the previous tag with left. The focus indicator loops, so when you are at the last option and press right it will focus on the first tag.
+In the stacked version of tag-list, when there are hidden tags, you can navigate only through visible tags with arrow keys. The first tag indicates how many hidden tags there are. Using a screen reader, it will announce how many hidden tags there are.
```

Comment:
We don't want to leave here an information that they can focus it with `Tab` key?